toolkit/tools/imagecustomizerapi/config.go

```diff
@@ -8,6 +8,7 @@ import "fmt"
 type Config struct {
 	Storage Storage `yaml:"storage"`
 	Iso     *Iso    `yaml:"iso"`
+	Pxe     *Pxe    `yaml:"pxe"`
```

Comment:
But we might have something in the future that's not optional. At a high level, the `pxe` node itself is optional since there are many scenarios that will not require it. Does this make sense?

Reply:
kept as is.